app/javascript/pages/getAssistance/get-assistance.tsx

```diff
@@ -1,15 +1,114 @@
-import { t } from "@bloom-housing/ui-components"
+import { ActionBlock, Icon, t, Heading } from "@bloom-housing/ui-components"
+import Link from "../../navigation/Link"
 import React from "react"
 import AssistanceLayout from "../../layouts/AssistanceLayout"
 import withAppSetup from "../../layouts/withAppSetup"
+import {
+  faClipboardList,
+  faHouseChimney,
+  faDoorOpen,
+  faPeopleGroup,
+} from "@fortawesome/free-solid-svg-icons"
+import { faYoutube } from "@fortawesome/free-brands-svg-icons"
+import bloomTheme from "../../../../tailwind.config"
 
 const GetAssistance = () => {
   return (
     <AssistanceLayout
       title={t("assistance.title.getAssistance")}
       subtitle={t("assistance.subtitle.getAssistance")}
     >
-      <h1>{t("assistance.title.getAssistance")}</h1>
+      <ActionBlock
```

Comment:
This all looks great! One potential item to consider: if the resources on this page would change in the future (add a link, etc) it may be advantageous to use a map function that iterates over a json dictionary/array for these items.